Authenticator/Source/TokenFormViewController.swift

```diff
@@ -237,19 +234,19 @@ extension TokenFormViewController {
             return cell
 
         case .TokenTypeRow(let viewModel):
-            let cell = tableView.dequeueReusableCellWithClass(TokenTypeRowCell.self)
+            let cell = tableView.dequeueReusableCellWithClass(SegmentedControlRowCell<Form.Action>.self)
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 104 characters (line_length)